src/main/java/org/spongepowered/common/mixin/core/entity/player/MixinEntityPlayerMP.java

```diff
@@ -629,6 +649,32 @@ public void setImplVelocity(Vector3d velocity) {
     }
 
     @Override
+    public Map<Statistic, Long> getStatistics() {
+        StatisticsFile statisticsFile = statsFile;
```

Comment:
Is there a reason why you store the field in a variable instead of casting it directly?